src/main/java/com/zepben/evolve/services/common/translator/BaseProtoToCim.kt

```diff
@@ -49,8 +53,18 @@ fun toCim(pb: PBIdentifiedObject, cim: IdentifiedObject, baseService: BaseServic
         name = pb.name.internEmpty()
         description = pb.description.internEmpty()
         numDiagramObjects = pb.numDiagramObjects
+        pb.namesList.forEach { addName(toCim(it, baseService)) }
     }
 
+fun toCim(pb: PBNameType): NameType =
+    NameType(pb.name).apply {
+        description = pb.description
+    }
+
+fun toCim(pb: PBName, baseService: BaseService): Name =
```

Comment:
And this is where we have a problem with sending models over the wire. I think in this case I'm happy for name types to be created based purely on their type name. When we add the ability to pull name types back over grpc, if one already exists in the service, we just update it's description with the one that comes back from the server.